code/game/objects/items/storage/boxes.dm

```diff
@@ -643,8 +638,13 @@
 /obj/item/storage/box/nade_box/update_icon()
 	if(!length(contents))
 		icon_state = "[base_icon]_e"
+		overlays.Cut()
```

Comment:
The overlays need to be cut every call, not just when empty. Every time you add/remove a grenade this will be called, so the overlays just keep adding up. 
![overlays](https://github.com/user-attachments/assets/9474e086-27dd-4c03-ae72-5e47b601afc2)

Reply:
honestly i forgot that it's needed after using mutable overlays on other servers, fixed